README.md

```diff
@@ -1,14 +1,31 @@
+As of May 2018, only three applications are using NYPLBase: Locations App (previous Location-prototype), Research Collections, and Staff Profiles. Some of the older Drupal pages also use NYPLBase. Subsequent applications are using NYPL Design Toolkit, with the idea that NYPL Design Toolkit will supersede NYPLBase as the default.
 
-The idea is that anyone who's building a web project for the Library, whether one of our internal dev teams, or an external consultant group, can download and use this code as the style base, and not have to worry about getting our brand right--it's already in there. Even if you're not using the code, you can still look at the demo page as a style guide. (Eventually NYPLBase will be part of a much larger branding resource. Stay tuned!)
+## Usage ##
+To update the styles for the apps that use NYPLBase, simply make a feature branch and make the changes. Then you need to use SASS to compile the changes to `min` files.
 
-As far as I know, the Library has never had a resource like this. So, it's a bit of an experiment! I'm hoping that because the style guide is written in HTML/CSS, it will not only make prototyping faster and more accurate, but it will evolve and also adapt to other, similar kinds of code, such as iOS or Android app code. 
+Please run
+`sass css/style.scss:css/nyplbase.min.css --style compressed`. It should generate the min CSS and min CSS map under the name `nyplbase`.
 
-I have already started using NYPLBase myself to build prototypes of new designs for the site. As I build, I'm working directly in the code for Base so that I can make fixes to the core as I go. 
+Please go to the particular project's documentation for more details about testing the styles.
 
-At the moment, I'm the only developer allowed to work like this. At some point I will start accepting pull requests on GitHub from Trusted Colleagues.
\ No newline at end of file
+Locations App - https://github.com/NYPL/locations-app
+
+## NYPL Deployment Proces s##
+The process will need the access to NYPL's `UX-Static` repo and Jenkins accounts.
+
+After the changes are approved, we need to update the changes to the repo of `UX-Static`.
+
+https://bitbucket.org/NYPL/ux-static/src/master/
+
+In `UX-Static`, find the folder named, `nyplbase`. Under `nyplbase`, you will see the folders of different versions. Create a new folder with the new version. In each folder, there are three different folders that contain images, fonts, and CSS stylesheets. Copy the contents from the folder of previous version to the new folder that we just created. And in the CSS folder, paste the CSS min files we compiled from NYPLBase. Push the changes. If you have updates for fonts and images, copy them to the new folders respectively as well.
```

Comment:
No comma needed after `named`
Change "three different folders" to "three folders"
Change "that we just created" to "that was just created"